addons/account/views/report_payment_receipt_templates.xml

```diff
@@ -20,7 +21,9 @@
                                 Vendor:
                             </t><span t-field="o.partner_id"/>
                         </div>
-                        <div class="col-6" t-if="o.payment_method_id">
+                        <div name="payment_method"
+                             t-if="values['display_payment_method'] and o.payment_method_id"
```

Comment:
where is _get_payment_receipt_report_values overwritten to add `display_payment_method` and `display_invoices` ?

:eyes: